lib/sysopia.rb

```diff
@@ -1,4 +1,3 @@
-require_relative "../environment"
```

Comment:
the intent of getting environment loaded here is decoupling of models and library from presentation layer (helpers, 'controller'  and views. However as this project is practically all about presentation -- I think it is ok to couple them as you did, so it is fine as you did